assignment6/rearranging_cars_test.py

```diff
@@ -0,0 +1,26 @@
+'''
+Created on 07 July 2017
+
+@author: Eva
+'''
+
+import unittest
+from rearranging_cars import rearrange_cars, Move
+
+class UnknownLanguageTest(unittest.TestCase):
```

Comment:
Oops! Have changed it now, but I have not uploased it as I would like to add a few more tests :)